### PR TITLE
server: Use event trigger name as tracing span name

### PR DESCRIPTION
**Pull request by paf31**
# Description

This changes the tracing span name for event triggers and scheduled triggers to use the trigger name, for easier debugging.

Fixes https://github.com/hasura/graphql-engine-pro/issues/512

### Changelog

- [x] `CHANGELOG.md` is updated with user-facing content relevant to this PR. If no changelog is required, then add the `no-changelog-required` label.

### Affected components

- [x] Server

### Related Issues

https://github.com/hasura/graphql-engine-pro/issues/512

### CODE REVIEW

File: server/src-lib/Hasura/Eventing/EventTrigger.hs [hunk 1]
```diff
@@ -264,7 +265,7 @@ processEventQueue logger logenv httpMgr pool getSchemaCache eeCtx@EventEngineCtx
             -- For such an event, we unlock the event and retry after a minute
             setRetry e (addUTCTime 60 currentTime)
           >>= flip onLeft logQErr
-        Just eti -> do
+        Just eti -> runTraceT ("Event trigger: " <> unNonEmptyText (unTriggerName (etiName eti))) do
```
Comment (paf31) [inline server/src-lib/Hasura/Eventing/EventTrigger.hs:268]: I'm not sure I need the prefix `Event trigger: ` here really. What do people think?

Comment (0x777) [reply server/src-lib/Hasura/Eventing/EventTrigger.hs:268]: I wouldn't mind seeing this if I'm debugging something.

**Review by 0x777 — APPROVED** [review]

**Comment by tirumaraiselvan** [comment]
@paf31 Do you mind including this review comment in this PR: https://github.com/hasura/graphql-engine/pull/5718/files#r482888343

**Review by tirumaraiselvan — APPROVED** [review]

**Review by evertedsphere — APPROVED** [review]
lgtm :+1:

**Pull request merged**